app/views/layouts/blacklight/base.html.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<%= content_tag :html, class: 'no-js h-100', **html_tag_attributes do %>
+<%= content_tag :html, class: 'h-100 no-js', **html_tag_attributes do %>
```

Comment:
The "no-js" seems to be removed by script a few lines down, what's it being added for if it's just removed?

Reply:
If the user's browser has JavaScript turned off, then they will get the message inside the "no-js" div because it can't be removed by JavaScript.